kerchunk/hdf.py

```diff
@@ -302,3 +304,10 @@ def _storage_info(self, dset: h5py.Dataset) -> dict:
                     [a // b for a, b in zip(blob.chunk_offset, chunk_size)])
                 stinfo[key] = {'offset': blob.byte_offset, 'size': blob.size}
             return stinfo
+
+
+def _is_netcdf_datetime(dataset: h5py.Dataset):
+    attrs = dataset.attrs
+    # This is the same heuristic used by xarray
+    # https://github.com/pydata/xarray/blob/f8bae5974ee2c3f67346298da12621af4cae8cf8/xarray/coding/times.py#L670
+    return "units" in attrs and "since" in attrs["units"]
```

Comment:
Should be `attrs.get("units", "")`, since the attribute might not exist.

Reply:
Shouldn't the initial `"units" in attrs` handle that case?

Reply:
Sorry, I'm talking rubbish, ignore me :)

Reply:
No worries :-)